static/app/components/onboarding/consoleModal.tsx

```diff
@@ -91,35 +64,20 @@ const consoleConfig = {
           }
         )}
       </p>
-      <p>{t('Support for Switch 2 as well as the original Switch is available.')}</p>
+      <p>{t('Sentry supports both the original Switch and Switch 2.')}</p>
     </Fragment>
   ),
   xbox: (
     <Fragment>
       <p>
-        {tct(
-          "You can get started using Sentry by [microsoftGameDevelopmentKitLink: requesting Microsoft Game Development Kit (GDK) Middleware access]. We'll receive your request and get back to you with the next steps.",
-          {
-            microsoftGameDevelopmentKitLink: (
-              <ExternalLink href="https://developer.microsoft.com/en-us/games/support/request-gdkx-middleware" />
-            ),
-          }
-        )}
-      </p>
-      <p>
-        {t(
-          'Support is available on Xbox Series X and S, by adding the Sentry SDK to your game. Crash collection as well as non-fatal events can be captured, on devkits as well as retail devices.'
-        )}
+        {t('Sentry supports Xbox Series X and S, devkits as well as retail devices.')}
```

Comment:
We also support Xbox One now. How about:

```suggestion
        {t('Sentry supports Xbox One and Series X|S, devkits as well as retail devices.')}
```

I often see Xbox materials using X|S to refer to both.